.ci/pytorch/test.sh

```diff
@@ -691,12 +691,14 @@ test_inductor_get_core_number() {
 
 test_inductor_set_cpu_affinity(){
   #set jemalloc
-  JEMALLOC_LIB="$(find /usr/lib -name libjemalloc.so.2)"
-  IOMP_LIB="$(dirname "$(which python)")/../lib/libiomp5.so"
-  export LD_PRELOAD="$JEMALLOC_LIB":"$IOMP_LIB":"$LD_PRELOAD"
-  export MALLOC_CONF="oversize_threshold:1,background_thread:true,metadata_thp:auto,dirty_decay_ms:-1,muzzy_decay_ms:-1"
-  export KMP_AFFINITY=granularity=fine,compact,1,0
-  export KMP_BLOCKTIME=1
+  if [[ "${TEST_CONFIG}" != *aarch64* ]]; then
+    JEMALLOC_LIB="$(find /usr/lib -name libjemalloc.so.2)"
```

Comment:
Can you please move JEMALLOC and MALLOC_CONFG outside  of that loop?

Reply:
I wasn't sure if libjemalloc exists on aarch64 CI. I can add it back in another PR.